test/ctia/stores/es/init_test.clj

```diff
@@ -32,8 +34,9 @@
 
 (deftest init-store-conn-test
   (testing "init store conn should return a proper conn state with unaliased conf"
-    (let [{:keys [index props config conn]}
-          (sut/init-store-conn props-not-aliased)]
+    (let [services (->ESConnServices)
```

Comment:
There will be no `app` in this namespace because there are no `fixture-ctia` fixtures to start CTIA. Thus we use `->ESConnServices` over `app->ESConServices`.